slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -164,16 +164,19 @@ async def monitor_current_session(self) -> None:
                         )
                     break
                 try:
-                    # The logging here is for detailed trouble shooting of potential issues in this client.
-                    # If you don't see this log for a while, it means that
-                    # this receive_messages execution is no longer working for some reason.
-                    counter_for_logging = (counter_for_logging + 1) % logging_interval
-                    if counter_for_logging == 0:
-                        log_message = (
-                            f"{logging_interval} session verification executed after the previous same log"
-                            f" ({session_id})"
-                        )
-                        self.logger.debug(log_message)
+                    if self.trace_enabled and self.logger.level <= logging.DEBUG:
+                        # The logging here is for detailed investigation on potential issues in this client.
+                        # If you don't see this log for a while, it means that
+                        # this receive_messages execution is no longer working for some reason.
+                        counter_for_logging = (
+                            counter_for_logging + 1
+                        ) % logging_interval
+                        if counter_for_logging == 0:
+                            log_message = (
+                                f"{logging_interval} session verification executed after the previous same log"
```

Comment:
> Session verification message received

This suggested log message may not be accurate. In this method, all we do is just to monitor the current session is active. No communication with the server-side is not involved here (ping-pong comms are done by aiohttp internals but it's not in this method).

Also, the intention of logging here is to periodically print some logs to tell the connection is alive even when the connection has not been receiving any incoming requests from Slack for a while. For this reason, I would like to avoid accumulating the count, which wont' be used for any purposes.

To make the log message clearer, I may want to revise it like:

> monitor_current_session method has verified the session is active (session: {session_id}, logging interval: {logging_interval})

Reply:
Thanks for taking the time to write a detailed explanation, @seratch. Your proposed wording sounds good to me, and is an improvement (in my opinion) over the original message. (Having the `{logging_interval}` at the beginning didn't make sense to me until I checked the code to see what it was indicating.)